tests/unit/systems/dag/runtimes/test_triton.py

```diff
@@ -77,3 +89,54 @@ def test_triton_runtime_export_and_run(runtime, model_name, expected_model_name,
     assert response is not None
     # assert isinstance(response, DictArray)
     assert len(response["filtered_ids"]) == 80
+
+
+@pytest.mark.skipif(not TRITON_SERVER_PATH, reason="triton server not found")
+@pytest.mark.parametrize(
+    ["runtime", "model_name", "expected_model_name"],
+    [
+        (None, None, "ensemble_model"),
+        (TritonEnsembleRuntime(), None, "ensemble_model"),
+        (TritonEnsembleRuntime(), "triton_model", "triton_model"),
+        # (TritonExecutorRuntime(), None, "executor_model"),
+        # (TritonExecutorRuntime(), "triton_model", "triton_model"),
+    ],
+)
+def test_workflow_with_forest_inference(runtime, model_name, expected_model_name, tmpdir):
```

Comment:
I'd suggest splitting the specific op tests out into separate files. Maybe a directory like `tests/unit/systems/dag/runtimes/triton/ops`?